### DISCUSSION TITLE
Why I can't use controls in FluentAvalonia?

### DISCUSSION THREAD

**Discussion by akchiji888** [amwx/FluentAvalonia#377] [Q&A]
I have imported FluentAvalonia into my projects successfully,but I can't use controls in FluentAvalonia.
This is my XMAL file:
![image](https://github.com/amwx/FluentAvalonia/assets/92619820/e23e0f82-0c43-4449-97bf-fa30630c2653)
And this is the error message:
![image](https://github.com/amwx/FluentAvalonia/assets/92619820/bf77eea7-ea3c-4a13-877e-59efad53125e)

**Comment by robloo** [comment]
You don't even have a button in your XAML so it doesn't make sense to me. However, it works for other projects so it's something wrong with your configuration.

Do you have something similar to the below configured in App.xaml?

```xaml
<Application
    xmlns="https://github.com/avaloniaui"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:fas="using:FluentAvalonia.Styling"
    x:CompileBindings="True">

    <Application.Styles>
        <fas:FluentAvaloniaTheme
            TextVerticalAlignmentOverrideBehavior="EnabledNonWindows" />
    </Application.Styles>

</Application>

```

 If so you can use it in other views just as you are doing.

```xaml
<ResourceDictionary
    xmlns="https://github.com/avaloniaui"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:fac="using:FluentAvalonia.UI.Controls">
    <ControlTheme></ControlTheme>
</ResourceDictionary>
```

**Comment by timunie** [comment]
@akchiji888 if the issue persists, please provide a full minimal sample to reproduce.
